backend/package.json

```diff
@@ -9,8 +9,8 @@
     "lintfix": "eslint --fix .",
     "start": "babel-node server",
     "mockTest": "bash test.sh",
-    "dockerTest": "mocha tests --recursive --require babel-polyfill --require @babel/register",
-    "hostedTest": "mocha tests --recursive --require babel-polyfill --require @babel/register"
+    "dockerTest": "mocha tests/tests --require babel-polyfill --require @babel/register",
```

Comment:
Still a little unsure about having two scripts being the same with different names  (`hostedTest`, `dockerTest`)

Reply:
I will just remove the hosted option.